rerun_cpp/CMakeLists.txt

```diff
@@ -125,6 +148,7 @@ install(TARGETS rerun_sdk
     EXPORT rerun_sdkTargets
     LIBRARY DESTINATION lib
     ARCHIVE DESTINATION lib
+    RUNTIME DESTINATION bin
```

Comment:
FYI I added this here just for consistencies, since CMake 3.14 these argument have default values that should just work fine, see https://cmake.org/cmake/help/v3.16/command/install.html#installing-targets .